integration/index_test.go

```diff
@@ -220,3 +222,11 @@ func TestSociIndexRemove(t *testing.T) {
 		}
 	})
 }
+
+func sociIndexFromDigest(sh *shell.Shell, indexDigest string) (index soci.Index, err error) {
+	rawSociIndexJSON := sh.O("soci", "index", "info", indexDigest)
+	if err = json.Unmarshal(rawSociIndexJSON, &index); err != nil {
+		err = fmt.Errorf("invalid soci index from digest %s: %s", indexDigest, err)
+	}
+	return
+}
```

Comment:
Could you move this out to testutil and refactor TestSociIndexInfo to use this? Will save work in the future redundancy reducing refactor.

Reply:
Will refactor `TestSociIndexInfo` to use this. But I would prefer to keep it in `index_test.go` since it's more relevant. Right now `testutil.go` is more for environment setup (shell, containerd, snapshotter, etc) (and already really long :)).

Personally I think we should have some `testutil_ztoc.go`/`testutil_index.go` to host ztoc/index related test logics. That should be a separate refactor and before that we can keep the relevant helpers in more relevant `*_test.go` for easy discovery

Reply:
fixed